test/tests/concrete_moisture_heat_transfer/tests

```diff
@@ -9,7 +9,7 @@
     rel_err = 1e-6
     abs_zero = 1e-10
   []
-  issues = '#245'
```

Comment:
I'd just keep issue 245 here. The 272 issue isn't really relevant.

Reply:
I have modified it now.